app/charts/shared/containers.tsx

```diff
@@ -69,13 +69,14 @@ export const ChartSvg = ({ children }: { children: ReactNode }) => {
       style={{ position: "absolute", left: 0, top: 0 }}
     >
       {interactiveFiltersConfig?.calculation.active && (
-        <Box
-          component="foreignObject"
+        <foreignObject
           width={width - margins.right}
-          sx={{ display: "flex", textAlign: "right", height: 26 }}
+          y={20}
+          height="26"
+          style={{ display: "flex", textAlign: "right" }}
         >
           <CalculationToggle />
-        </Box>
+        </foreignObject>
```

Comment:
@noahonyejese I am not sure if we need to revert this, I think the previous implementation missed `y: [18, 20]` in `sx`? 😄

Reply:
```javascript
No overload matches this call.
  Overload 1 of 2, '(props: { component: ElementType<any>; } & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<any> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>): Element', gave the following error.
    Type '{ display: "flex"; textAlign: "right"; y: number[]; height: number; }' is not assignable to type 'SxProps<Theme> | undefined'.
      Types of property 'y' are incompatible.
        Type 'number[]' is not assignable to type 'SystemStyleObject<Theme> | CssVariableType | ((theme: Theme) => string | number | SystemStyleObject<Theme>) | undefined'.
          Type 'number[]' is not assignable to type 'CSSSelectorObjectOrCssVariables<Theme>'.
            Index signature for type 'string' is missing in type 'number[]'.
  Overload 2 of 2, '(props: DefaultComponentProps<BoxTypeMap<{}, "div">>): Element', gave the following error.
    Type '{ display: "flex"; textAlign: "right"; y: number[]; height: number; }' is not assignable to type 'SxProps<Theme> | undefined'.
      Types of property 'y' are incompatible.
        Type 'number[]' is not assignable to type 'SystemStyleObject<Theme> | CssVariableType | ((theme: Theme) => string | number | SystemStyleObject<Theme>) | undefined'.ts(2769)
Box.d.ts(11, 7): The expected type comes from property 'sx' which is declared here on type 'IntrinsicAttributes & { component: ElementType<any>; } & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<...> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>'
Box.d.ts(11, 7): The expected type comes from property 'sx' which is declared here on type 'IntrinsicAttributes & SystemProps<Theme> & { children?: ReactNode; component?: ElementType<any> | undefined; ref?: Ref<...> | undefined; sx?: SxProps<...> | undefined; } & CommonProps & Omit<...>'
(property) sx?: SxProps<Theme> | undefined
```

I can't add the array into sx

Reply:
Yeah, looks like it's not supported for `y` :/